charts/sonarqube-dce/templates/sonarqube-application.yaml

```diff
@@ -283,8 +283,7 @@ spec:
               - -c
               - |
                 host="$(hostname -i || echo '127.0.0.1')"
-                reply=$(wget --no-proxy -qO- --header="X-Sonar-Passcode: $SONAR_WEB_SYSTEMPASSCODE" http://${host}:{{ .Values.service.internalPort }}{{ .Values.ApplicationNodes.livenessProbe.sonarWebContext }}api/system/liveness 2>&1)
-                if [ -z "$reply" ]; then exit 0; else exit 1; fi
+                wget --no-proxy --quiet -O /dev/null --timeout={{ .Values.livenessProbe.timeoutSeconds }} --header="X-Sonar-Passcode: $SONAR_WEB_SYSTEMPASSCODE" "http://${host}:{{ .Values.service.internalPort }}{{ .Values.livenessProbe.sonarWebContext }}api/system/liveness"
```

Comment:
Tests are failing for the templating on that line. you have a small typo, it should be `.Values.ApplicationNodes.livenessProbe.timeoutSeconds` instead of `.Values.livenessProbe.timeoutSeconds`

For dce edition we have a subkey for each component, either ApplicationNodes or searchNodes.

Reply:
```suggestion
                wget --no-proxy --quiet -O /dev/null --timeout={{ .Values.ApplicationNodes.livenessProbe.timeoutSeconds }} --header="X-Sonar-Passcode: $SONAR_WEB_SYSTEMPASSCODE" "http://${host}:{{ .Values.service.internalPort }}{{ .Values.livenessProbe.sonarWebContext }}api/system/liveness"
```